pubsub/cloud-client/iam.py

```diff
@@ -63,20 +63,17 @@ def set_topic_policy(project, topic_name):
     policy = client.get_iam_policy(topic_path)
 
     # Add all users as viewers.
-    policy.bindings.add(
-        role='roles/pubsub.viewer',
-        members=['allUsers'])
+    policy.bindings.add(role="roles/pubsub.viewer", members=["allUsers"])
 
     # Add a group as a publisher.
     policy.bindings.add(
-        role='roles/pubsub.publisher',
-        members=['group:cloud-logs@google.com'])
+        role="roles/pubsub.publisher", members=["group:cloud-logs@google.com"]
```

Comment:
Did we change the style for Python?

Reply:
@busunkim96 I used `black` which automatically changed single quotes to double. Should I change back?